File: chatwoot/ruby/src/DeleteCustomAttributeFromAccountExample.rb
require "json"
require "chatwoot_client"

ChatwootClient.configure do |config|
    config.api_key["api_access_token"] = "USER_API_KEY"
end

begin
    ChatwootClient::CustomAttributesApi.new.delete_custom_attribute_from_account(
        0, # account_id
        0, # id
    )
rescue ChatwootClient::ApiError => e
    puts "Exception when calling CustomAttributesApi#delete_custom_attribute_from_account: #{e}"
end

File: chatwoot/php/src/DeleteCustomAttributeFromAccountExample.php
<?php

namespace OSEG\ChatwootExamples;

require_once __DIR__ . '/../vendor/autoload.php';

use SplFileObject;
use Chatwoot;

$config = Chatwoot\Client\Configuration::getDefaultConfiguration();
$config->setApiKey("api_access_token", "USER_API_KEY");

try {
    (new Chatwoot\Client\Api\CustomAttributesApi(config: $config))->deleteCustomAttributeFromAccount(
        account_id: 0,
        id: 0,
    );
} catch (Chatwoot\Client\ApiException $e) {
    echo "Exception when calling CustomAttributesApi#deleteCustomAttributeFromAccount: {$e->getMessage()}";
}
